src/tty/mod.rs

```diff
@@ -30,10 +30,18 @@ mod windows;
 #[cfg(windows)]
 pub use self::windows::*;
 
+/// This struct contains information about the child process.
+pub struct ChildInfo {
+    #[cfg(unix)]
+    pub pid: libc::pid_t,
+}
```

Comment:
I feel like this could be useful on any platform, but since `spawn_new_instance` currently only works on 'nix anyway, I thought I'd leave it alone for now.